chaos/cases/cases.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"path/filepath"
+
+	"golang.org/x/sync/errgroup"
+
+	config2 "github.com/pingcap/dm/dm/config"
+	"github.com/pingcap/dm/dm/pb"
+	"github.com/pingcap/dm/pkg/utils"
+)
+
+var (
+	// NOTE: items in `doSchemas` should be specified in the corresponding task files (`filenames`).
+	// NOTE: can not support optimistic shard DDL now because go-sqlsmith will generated some statements like
```

Comment:
how about change it to `TODO` to remind us in future

Reply:
changed in https://github.com/pingcap/dm/pull/1114/commits/91e5cf704830dbd3fc88d361a660f6b958f43b4d